perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -22,6 +22,7 @@
 from perfkitbenchmarker import configs
 from perfkitbenchmarker import flags
 from perfkitbenchmarker import sample
+from perfkitbenchmarker import vm_util
 import re
 
 flags.DEFINE_boolean('ping_also_run_using_external_ip', False,
```

Comment:
Per conversation, please use the ShouldRunOnExternalIpAddress() style.

Reply:
Changed to use the style we discussed

Reply:
Please update the breaking changes section with the deleted flag and new usage.  Otherwise we'll let @cwilkes take one last look.

Reply:
I've added an entry in the breaking changes section